tests/conftest.py

```diff
@@ -109,10 +109,11 @@ def project():
     base_dir = ape.config.PROJECT_FOLDER
     contracts_dir = base_dir / "contracts"
     dependency_contracts_dir = base_dir / "bar" / "contracts"
-    contracts_dir.mkdir(exist_ok=True, parents=True)
+    sub_contracts_dir = contracts_dir / "subcontracts"
+    sub_contracts_dir.mkdir(exist_ok=True, parents=True)
     dependency_contracts_dir.mkdir(exist_ok=True, parents=True)
 
-    make_source(contracts_dir, "foo", FOO_SOURCE_CODE)
+    make_source(sub_contracts_dir, "foo", FOO_SOURCE_CODE)
```

Comment:
`foo` contracts gets added to sub dir, meaning without last changes, it would verify with a differen tsource Id (with prefix) and the test would have failed.

the test passes the same as it did before, showing we can no include subdirs and it will use the source ID Etherscan expects